app/services/organizations/create_service.rb

```diff
@@ -73,8 +73,7 @@ def create_user(email, first_name, last_name)
         email: email,
         first_name: first_name,
         last_name: last_name,
-        password: SecureRandom.hex(3)[0, 6],
-        tos_agreement: 1
+        password: SecureRandom.hex(3)[0, 6]
```

Comment:
Is it possible to create this user now we have the validation on User? Or does this create_user fail? Did you test creating an Organization?

Reply:
Ah yes, per this line `This check is performed only if terms_of_service is not nil.`